submissions/chilubaFungo/html/contact.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Contact Form Page</title>
+</head>
+<body>
+	<h2>For more information, kindly get in touch with us by filling in the form below</h2>
+		<form action="#" method="POST">
```

Comment:
Well done @Luuba . You need to remove the attributes `action` and `method` from the form tag.